extension-style/src/main/java/com/mapbox/maps/extension/style/sources/generated/GeoJsonSource.kt

```diff
@@ -649,11 +638,12 @@ class GeoJsonSource(builder: Builder) : Source(builder.sourceId) {
      *
      * @return the GeoJsonSource
      */
+    @SuppressWarnings("deprecation")
```

Comment:
No need to `SuppressWarning` imo.

Reply:
Without `@Suppress("DEPRECATION_ERROR") `we get `compiler error Using 'constructor GeoJsonSource(GeoJsonSource.Builder)' is an error. Use builder instead`